historical-frontend/choose.html

```diff
@@ -5,7 +5,7 @@
 
 {% include_relative _heading.html %}
 
-<h1 class="heading-medium">Choose a version to start with</h1>
```

Comment:
I think having a title that specifies that you have to do something is quite a strange pattern for GOV.UK* - To be honest I thought it was a hold over from when we had a more transactional flow. I'll have another look at it when I've cleared some stuff.

* I appreciate the irony of this given how the diff view has no grounding in existing patterns.